### ISSUE TITLE
How to Handle us-va-augusta Filename Changes?

### ISSUE THREAD

**Issue by waldoj** [openaddresses/openaddresses#977]
[We provide a filename of `Augusta_Address_Points_080514.zip`](https://github.com/openaddresses/openaddresses/blob/master/sources/us-va-augusta.json) for Augusta, VA (USA)'s address points file. But that's a snapshot file (in `MMDDYY` format) that is removed and replaced with a new file periodically, so every run fails to gather new data, and only cached data from last August is available. [At the moment](ftp://www.co.augusta.va.us/GIS/) the filename is `Augusta_Parcels_012015.zip`. It is the only file in the FTP site’s `/GIS/` directory of the format `Augusta_Parcels_*`. Does the project have a standard method of handling this?

**Comment by geobrando** [comment]
@waldoj I don't believe so.  Should be easy enough to incorporate into Machine's [`openaddr.cache()`](https://github.com/openaddresses/machine/blob/f5e97590d89d246657b44ec337696ae2b5d2ce6e/openaddr/cache.py) as [requests-ftp](https://github.com/Lukasa/requests-ftp) implements the `LIST` command which you could combine with some glob/regex pattern specified in the source json if necessary.

But I'm wondering if this is a rare use case and this particular case could be solved by contacting the county and asking that they save the date information for the metadata XML.

However looking at the current files the address point layer is gone and the parcel layer at ftp://www.co.augusta.va.us/GIS/Augusta_parcels_Aug2015.zip does not have addresses.  Think an address point layer will need to be requested again.

**Comment by waldoj** [comment]
Ugggghh. Thanks for checking on that. :-/ Next month, I'm keynoting Virginia's annual conference of county GIS employees—I'll work this into my talk.
